src/Parsers/ParserDropQuery.cpp

```diff
@@ -2,7 +2,7 @@
 
 #include <Parsers/CommonParsers.h>
 #include <Parsers/ParserDropQuery.h>
-
+#include <Parsers/ParserCreateQuery.h>
```

Comment:
why?

Reply:
This is because `ParserNameList` is defined in `ParserCreateQuery .h`, and I am not sure whether it is necessary to separate it.